frontend/website/src/pages/LeaderboardPage.re

```diff
@@ -75,15 +146,18 @@ let reducer = (_, action) => {
 [@react.component]
 let make = (~lastManualUpdatedDate) => {
   let (state, dispatch) = React.useReducer(reducer, initialState);
-
+  let (username, setUsername) = React.useState(() => "");
```

Comment:
could we move the username state to be part of the component state record and add it to the reducer? open to thoughts about this! :)